src/elements/common/sub-header/Add.js

```diff
@@ -20,21 +19,23 @@ type Props = {
 };
 
 const Add = ({ onUpload, onCreate, showUpload = true, showCreate = true }: Props) => (
```

Comment:
this might need investigation since `SubHeaderRight` passes `isDisabled`. I wonder if there's been a bug in here for a while